feature-libs/storefinder/components/store-finder-list-item/store-finder-list-item.component.html

```diff
@@ -38,6 +38,14 @@
         {{ location.formattedDistance }}
       </div>
     </div>
+    <ng-template
+      [cxOutlet]="'cx-pick-up-in-store-make-my-store'"
```

Comment:
Maybe create an enum for the outlets in `root`.

Reply:
Done thanks